textdb/textdb-gui/js/app.js

```diff
@@ -0,0 +1,51 @@
+var main = function(){
+		
+    $('.icon-menu').click(function(){
+        $('.menu').animate({
+            'left': '0px'
+        }, 200);
+		
+		$('.icon-menu').css({
+			'visibility': 'hidden',
+			'pointer': 'default'
+		});
+		
+		$('#main-delete').animate({
+			'margin-right': '295px'
+		}, 200);
+		
+        $('body').animate({
+            'left': '285px'
+        }, 200);
+    });
+	
+    $('.icon-close').click(function(){
+        $('.menu').animate({
+            'left': '-285px'
+        }, 200);
+		
+		$('#main-delete').animate({'margin-right': '10px'}, 200, function(){
+			$('.icon-menu').css({'visibility': 'visible', 'pointer': 'pointer'});
+		});
+		
+        $('body').animate({
+            'left': '0px'
+        }, 200);
+    });
+	
+	$('.menu ul li').on('click', function() {
```

Comment:
Same comment as the earlier PR, don't use anonymous functions, put them in variables and pass them on.